src/Marketing/InstalledExtensions.php

```diff
@@ -152,6 +158,33 @@ protected static function get_facebook_extension_data() {
 		return $data;
 	}
 
+	/**
+	 * Get Pinterest extension data.
+	 *
+	 * @return array|bool
+	 */
+	protected static function get_pinterest_extension_data() {
+		$slug = 'pinterest-for-woocommerce';
+
+		if ( ! PluginsHelper::is_plugin_installed( $slug ) ) {
+			return false;
+		}
+
+		$data         = self::get_extension_base_data( $slug );
+		$data['icon'] = plugins_url( 'images/marketing/pinterest.svg', WC_ADMIN_PLUGIN_FILE );
+
+		if ( 'activated' === $data['status'] ) {
+			// TODO: Get onboarded status from pinterest plugin.
+			// TODO: Finalise settings url.
+			// TODO: Finalise docs url.
+			$data['status']      = 'configured';
+			$data['settingsUrl'] = admin_url( 'admin.php?page=pinterest-for-woocommerce-setup-guide' );
+			$data['docsUrl']     = 'https://docs.woocommerce.com/document/pinterest-for-woocommerce/';
```

Comment:
I realise that according to the comment the docs url is not final, but it would still be going to include `utm_medium=product` as we're expected to do that for all outgoing links to wc com domain from now on. If the url is updated later hopefully this will remind the person who updates it.

Reply:
We should probably add for the other docs links in here. Happy to include in this PR if someone advises me to do so :)